tests/components/PropWithSymbol.vue

```diff
@@ -1,13 +1,7 @@
 <template>
-  <div>Symbol: {{ $props.sym }}</div>
+  <div>Symbol: {{ sym }}</div>
 </template>
 
 <script lang="ts" setup>
-interface Props {
-  sym?: Symbol
-}
-
-withDefaults(defineProps<Props>(), {
-  sym: () => Symbol()
-})
+defineProps({ sym: { type: Symbol, default: () => Symbol() } })
```

Comment:
Now I think about it, can't we keep the Ts syntax and use the destructuration props with default here?